pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -783,12 +784,28 @@ func (br *OVSBridge) GetOVSVersion() (string, Error) {
 		klog.Error("Transaction failed: ", err)
 		return "", NewTransactionError(err, temporary)
 	}
+
 	if len(res[0].Rows) == 0 {
-		klog.Warning("Could not find ovs_version")
-		return "", nil
+		klog.Warning("Could not find ovs_version in the OVS query result")
+		return "", NewTransactionError(fmt.Errorf("no results from OVS query"), false)
+	} else {
+		return ParseOvsVersion(res[0].Rows)
```

Comment:
Missing `[0]` here, "Rows" is a slice of `interface{}`. Now agent always failed to get ovs version:
```
E1029 03:30:40.133994       1 querier.go:109] Failed to get OVS client version: unexpected transaction result when querying OVSDB OVS version not found in ovsdb. Please configure your OVS (ovsdb) to provide version information.
E1029 03:30:41.126063       1 querier.go:109] Failed to get OVS client version: unexpected transaction result when querying OVSDB OVS version not found in ovsdb. Please configure your OVS (ovsdb) to provide version information.
E1029 03:30:42.129026       1 querier.go:109] Failed to get OVS client version: unexpected transaction result when querying OVSDB OVS version not found in ovsdb. Please configure your OVS (ovsdb) to provide version information.
E1029 03:30:43.128012       1 querier.go:109] Failed to get OVS client version: unexpected transaction result when querying OVSDB OVS version not found in ovsdb. Please configure your OVS (ovsdb) to provide version information.
```

Reply:
ah i guess this not covered in the unit tests?
how can i add a test for this?